helper/openapi/openapi.go

```diff
@@ -0,0 +1,102 @@
+// Package openapi provides structures and helpers that align with a subset of version 3 of the
+// OpenAPI specification: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md
+package openapi
+
+import (
+	"github.com/hashicorp/vault/version"
+)
+
+const OpenAPIVersion = "3.0.2"
+
+type Document struct {
+	OpenAPIVersion string               `json:"openapi"`
+	Info           Info                 `json:"info"`
+	Paths          map[string]*PathItem `json:"paths"`
+}
+
+type Info struct {
+	Title   string `json:"title"`
+	Version string `json:"version"`
+}
+
+type PathItem struct {
```

Comment:
s/PathItem/Path/

Reply:
One goal in this file was to align (where applicable) with the terms defined in OpenAPI, e.g. https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#path-item-object

Though given that Path doesn't exist elsewhere, I could see renaming it without loss of clarity.

Reply:
That makes sense, thanks for the context!